docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- Linux
+- a kernel with KVM enabled (`/dev/kvm` must exist)
+- at least `CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities
+- firecracker (v0.21.0 or higher)
+- `bridge`, and `firewall` CNI plugins from the [standard CNI plugins](https://github.com/containernetworking/cni), and `tc-redirect-tap` CNI plugin from the [Firecracker Go SDK](https://github.com/firecracker-microvm/firecracker-go-sdk/tree/master/cni) installed to `/opt/cni/bin`
+- iptables
+- `/etc/cni/conf.d` directory should exist
+- `/var/run/netns` directory should exist
+
+## Create the Cluster
+
+```bash
+sudo talosctl cluster create --wait --provisioner firecracker
+```
+
+Once the above finishes successfully, your talosconfig(`~/.talos/config`) will be configured to point to the new cluster.
+
+## Retrieve and Configure the `kubeconfig`
+
+```bash
+talosctl kubeconfig .
+kubectl --kubeconfig kubeconfig config set-cluster talos-default --server https://127.0.0.1:6443
+```
+
+## Using the Cluster
```

Comment:
suggestion, not sure if we need that, but we could add a few words about networking, that bridge interface `talosXYZ` is going to be created, assigned default IP 172.20.0.1, and each node will be directly accessible as well with 127.20.0.2-..., loadbalancer runs on 172.20.0.1 which does loadbalancing for Talos API and K8s API across master nodes?

(one can change default subnet used for provisioning)

Reply:
sorry, default network is `10.5.0.0/24` (see below), so LB is going to be 10.5.0.1, and nodes go 10.5.0.2 (like with docker)